sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -70,10 +63,28 @@ public IEnumerable<TestSuite> BuildFrom(ITypeInfo typeInfo, IPreFilter filter)
 
             if (liveVersions.Any())
             {
-                if (OnlyTestLatestServiceVersionLazy.Value)
+                if (TestEnvironment.GlobalTestOnlyLatestVersion)
                 {
                     _actualLiveServiceVersions = new[] { liveVersions.Max(Convert.ToInt32) };
                 }
+                else if (TestEnvironment.GlobalTestServiceVersions.Any())
+                {
+                    var enumType = _serviceVersions.First().GetType();
+                    var selectedVersions = new List<int>();
+
+                    foreach (var versionString in TestEnvironment.GlobalTestServiceVersions)
+                    {
+                        try
+                        {
+                            selectedVersions.Add(Convert.ToInt32(Enum.Parse(enumType, versionString)));
+                        }
+                        catch
+                        {
+                        }
```

Comment:
might be good to print out somewhere so that it shows up in logs in the pipeline. In case somebody wonders what's wrong.